object_class_method_variable/board.rb

```diff
@@ -1,50 +1,35 @@
 require 'faker'
-class Board
-  def initialize
-    @board_title = Faker::Color.color_name
-  end
-
-  def mark_favorite
-    # mark as favorite
-  end
-
-  def rename(new_name)
-    # rename board
-  end
+require_relative 'list'
 
-  def delete
-    #delete board
-  end
-
-  def add_list(list)
-    # create list on the board
-  end
-
-  def copy_list(list, new_list_name)
-    return new_list
-  end
+class Board
 
-  def rename_list(list, new_list_name)
-    # code here
-  end
+  attr_reader :board_title, :is_favorite, :is_closed
 
-  def move_list(dest_board, list, position)
-    # code here
+  def initialize
+    @board_title = Faker::Color.color_name.capitalize + ' Board'
```

Comment:
I would recommend to use @title
then you'll have

board.title = 'New title'
board.title

instead of board.board_title